pandas/tests/frame/test_subclass.py

```diff
@@ -211,6 +211,12 @@ def test_subclass_align_combinations(self):
         tm.assertIsInstance(res2, tm.SubclassedDataFrame)
         tm.assert_frame_equal(res2, exp1)
 
+    def test_subclass_iterrows(self):
+        # GH 13977
+        df = tm.SubclassedDataFrame({'a': [1]})
+        for i, row in df.iterrows():
+            tm.assert_series_equal(row, df.loc[i])
```

Comment:
Even though this is already tested in `test_indexing_sliced` which tests `df.loc`?